tests/test_api_perms.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
```

Comment:
Better name the file with the whole name `test_api_permissions.py` to be more readable?